dat/RPG Maker.dat

```diff
@@ -8,49 +8,49 @@ clrmamepro (
 game (
 	name ".flow"
 	description ".flow (ver.0.194)(English Patched)"
-	rom ( name RPG_RT.ini size 60192 crc 1204305322 md5 d89da02d37cc18fbe4224aeae13bfb76 sha1 df27d409e0b128dd6556e90677da67b016bcbbb9 )
+	rom ( name RPG_RT.ini size 60192 crc 47C83DAA md5 d89da02d37cc18fbe4224aeae13bfb76 sha1 df27d409e0b128dd6556e90677da67b016bcbbb9 )
 )
 
 game (
 	name "Dead Again"
 	description "Dead Again"
-	rom ( name RPG_RT.ini size 2019 crc 1429345850 md5 d0536abcfac9fe6588b8c81d1d7c7a93 sha1 3ec5c4aeb4a6d49aa8d7db01d7eb5f754adf683d )
+	rom ( name RPG_RT.ini size 2019 crc 5532163A md5 d0536abcfac9fe6588b8c81d1d7c7a93 sha1 3ec5c4aeb4a6d49aa8d7db01d7eb5f754adf683d )
 )
 
 game (
 	name "Dessert Nightmare"
 	description "Desert Nightmare (English Translation)"
-	rom ( name RPG_RT.ini size 2009 crc 4282449154 md5 130853a12cbe7fa33b8259c72b13fb36 sha1 9f04c16c591351bee98e2053565f5bdbffdd5845 )
+	rom ( name RPG_RT.ini size 2009 crc FF40FD02 md5 130853a12cbe7fa33b8259c72b13fb36 sha1 9f04c16c591351bee98e2053565f5bdbffdd5845 )
 )
 
 game (
 	name "Embric of Wulfhammer's Castle"
 	description "Embric of Wulfhammer's Castle"
-	rom ( name RPG_RT.ini size 2889959490 crc 2889959490 md5 c7e1be4e3f5c55bf2030dae1939be88e sha1 b830be3e44eba9a4387b4df80cb8ec2ed632c136 )
```

Comment:
This line looks suspect. Previously the size had the same value as the CRC. One of those has to be incorrect, but I don't know the upstream source is for this RPG Maker DAT to confirm.

Reply:
Hah, yeah there's no way the size matches the CRC value. The size doesn't really matter for the dat file. It could likely just be removed.